terraform/rtmp-server/bootstrap-post-dns.sh

```diff
@@ -17,15 +17,14 @@ systemctl stop nginx.service
 certbot certonly -n --agree-tos --standalone --email "${EMAIL_ADDRESS}" -d "${DOMAIN_NAME}" -d "ipfs-server.${DOMAIN_NAME}" -d "ipfs-gateway.${DOMAIN_NAME}"
 
 # Configure nginx with HTTPS
-cp -f /tmp/ipfs-server/nginx-default /etc/nginx/sites-available/default
-sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g" /etc/nginx/sites-available/default
+cp -f /tmp/rtmp-server/nginx-default  /usr/local/nginx/conf/conf.d/default
+sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g"  /usr/local/nginx/conf/conf.d/default
 
-cp -f /tmp/ipfs-server/nginx-gateway "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}"
-sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g" "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}"
-ln -s "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}" "/etc/nginx/sites-enabled/ipfs-gateway.${DOMAIN_NAME}"
-
-systemctl start nginx.service
+cp -f /tmp/rtmp-server/nginx-gateway "/usr/local/nginx/conf/conf.d/ipfs-gateway.${DOMAIN_NAME}"
+sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g" "/usr/local/nginx/conf/conf.d/ipfs-gateway.${DOMAIN_NAME}"
 
 # Configure auto-renewals
 echo "30 2 * * 1 certbot renew >> /var/log/letsencrypt/letsencrypt.log" >> /etc/crontab
-echo "35 2 * * 1 systemctl reload nginx" >> /etc/crontab
\ No newline at end of file
+echo "35 2 * * 1 systemctl reload nginx" >> /etc/crontab
+
+systemctl start nginx.service
```

Comment:
Why is this moved?

Reply:
Just added an nginx start, since we are not starting it as a process